library/core/src/num/int_macros.rs

```diff
@@ -1421,8 +1421,15 @@ macro_rules! int_impl {
         /// # Examples
         ///
         /// ```
-        #[doc = concat!("assert_eq!(0x1", stringify!($SelfT), ".unbounded_shl(4), 0x10);")]
-        #[doc = concat!("assert_eq!(0x1", stringify!($SelfT), ".unbounded_shl(129), 0);")]
+        #[doc = concat!("assert_eq!(0x1_", stringify!($SelfT), ".unbounded_shl(4), 0x10);")]
```

Comment:
annot: I use the `_` in them because that pattern helps catch mistakes via https://rust-lang.github.io/rust-clippy/master/index.html#mistyped_literal_suffixes (and I think it helps readability too).